elf_reader_test.go

```diff
@@ -131,6 +131,12 @@ func TestLoadCollectionSpec(t *testing.T) {
 				SectionName: "static",
 				License:     "MIT",
 			},
+			"anon_const": {
```

Comment:
Seems like there is no fix for the problem here, this means that the tests will fail on this commit? If yes, that's problematic when trying to bisect problems.

Reply:
Okay, didn't know we really cared about that, will respin the set to take this into account.

Reply:
Done.